packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -1,10 +1,30 @@
 import json
+import logging
 import uuid
 
+from functools import wraps
```

Comment:
`functools` is in the standard library, so this import should be grouped with the others at the top.